### DISCUSSION TITLE
Horizontal draggable tabs

### DISCUSSION THREAD

**Discussion by stevorr** [Zertz/react-headless-tabs#227] [Ideas]
Hi - I'm looking to combine the browser example and draggable example to get horizontal draggable tabs with add/delete functionality. Is this a simple thing to do?

Thanks!

**Comment by Zertz** [comment]
It should be possible to merge the functionality of both examples. You could clone the repo locally to tinker with them!

I'm not familiar with the details but most/all drag and drop libraries should be able to handle both orientations.
